lib/ui/painting.dart

```diff
@@ -1602,6 +1603,7 @@ class Canvas extends NativeFieldWrapperClass2 {
   /// Draws a sequence of points according to the given [PointMode].
   ///
   /// The `points` argument is interpreted as offsets from the origin.
+  @Deprecated('Let us know if you use this function. See <https://github.com/flutter/flutter/issues/10087>.')
```

Comment:
I would add more, namely, "We are planning on changing the signature this function." or some such. Otherwise people won't feel the motivation to tell us.